src/main/java/io/xdb/core/client/BasicClient.java

```diff
@@ -30,33 +27,7 @@ public BasicClient(final ClientOptions clientOptions) {
         this.defaultApi = buildDefaultApi();
     }
 
-    public String startProcess(
-        final String processType,
-        final String processId,
-        final String startStateId,
-        final Object input,
-        final BasicClientProcessOptions processOptions
-    ) {
-        final ProcessExecutionStartRequest request = new ProcessExecutionStartRequest()
-            .processType(processType)
-            .processId(processId)
-            .workerUrl(clientOptions.getWorkerUrl())
-            .startStateId(startStateId)
-            .startStateInput(clientOptions.getObjectEncoder().encode(input));
-
-        if (processOptions.getProcessOptionsOptional().isPresent()) {
-            final ProcessOptions options = processOptions.getProcessOptionsOptional().get();
-            request.processStartConfig(
-                new ProcessStartConfig()
-                    .idReusePolicy(options.getProcessIdReusePolicy())
-                    .timeoutSeconds(options.getTimeoutSeconds())
-            );
-        }
-
-        if (processOptions.getStartStateConfig().isPresent()) {
-            request.startStateConfig(processOptions.getStartStateConfig().get());
-        }
-
+    public String startProcess(final ProcessExecutionStartRequest request) {
```

Comment:
Use xdb public request as the input to BasicClient to reduce the SDK side wrapper.